src/applications/static-pages/download-1095b/components/App/index.js

```diff
@@ -117,90 +129,67 @@ export const App = ({ displayToggle, isLOA1, loggedIn, toggleLoginModal }) => {
     () => {
       getAvailableForms().then(result => {
         const mostRecentYearData = result[0];
-        if (
-          mostRecentYearData &&
-          mostRecentYearData?.lastUpdated &&
-          mostRecentYearData?.year
-        ) {
-          const date = new Date(mostRecentYearData.lastUpdated);
-          const options = { year: 'numeric', month: 'long', day: 'numeric' };
-          // expected output (varies according to local timezone and default locale): December 20, 2012
-          updateLastUpdated(date.toLocaleDateString(undefined, options));
```

Comment:
Date stuff no longer needed since the last updated date feature is no longer a part of the design.